ahcore/callbacks/tiff_callback.py

```diff
@@ -200,7 +200,7 @@ def _write_tiff(
             mpp=h5_reader.mpp,
             tile_size=tile_size,
             pyramid=True,
-            compression=TiffCompression.CCITTFAX4,
+            compression=TiffCompression.ZSTD,
```

Comment:
What is the reason for this change?

Reply:
Both @moerlemans and I noticed this error and I raised an issue earlier [here](https://github.com/NKI-AI/ahcore/issues/48). When you set it, did you try it with multiprocessing?